bz-mill/build/build

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+cat config/definitions/map_namespace_to_query |\
```

Comment:
this is not going to work because the definitions is not part of the job. For this to work the job itself would need to be cloned (you would need to define an scm section in the config) and then call into it.